src/frontends/paddle/tests/test_models/gen_scripts/generate_one_hot_v2.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# one_hot_v2 paddle model generator
+#
+import paddle
+import numpy as np
+from save_model import saveModel
+import sys
+
+
+def one_hot_v2_1(name: str, x, num_classes):
```

Comment:
Since there are some duplicated code, can we combine these 2 functions together? You can refer to ```top_k_v2```